client-registry-api/src/main/java/uk/gov/di/authentication/clientregistry/lambda/ClientRegistrationHandler.java

```diff
@@ -110,6 +114,27 @@ public APIGatewayProxyResponseEvent handleRequest(
                         });
     }
 
+    private ClientRegistrationRequest validateVectorsOfTrust(
+            ClientRegistrationRequest clientRegistrationRequest) {
+
+        if (clientRegistrationRequest.getVectorsOfTrust() == null)
+            clientRegistrationRequest.setVectorsOfTrust(
+                    AuthenticationValues.VERY_HIGH_LEVEL.getValue());
+
+        List<String> authenticationValues =
+                List.of(
+                        AuthenticationValues.LOW_LEVEL.getValue(),
+                        AuthenticationValues.MEDIUM_LEVEL.getValue(),
+                        AuthenticationValues.HIGH_LEVEL.getValue(),
+                        AuthenticationValues.VERY_HIGH_LEVEL.getValue());
+
+        if (!authenticationValues.contains(clientRegistrationRequest.getVectorsOfTrust()))
+            clientRegistrationRequest.setVectorsOfTrust(
+                    AuthenticationValues.VERY_HIGH_LEVEL.getValue());
```

Comment:
set this to medium as that is all that is supported for go live 
https://github.com/alphagov/digital-identity-architecture/pull/17/files

Reply:
do this all parts of the code base - medium is the highest that is currently supported